.github/workflows/macos.yml

```diff
@@ -100,8 +102,8 @@ jobs:
         run: |
```

Comment:
Previous suggestions also apply here, with the exception of `-DCMAKE_PREFIX_PATH`, which doesn't need to be set since we're installing Qt with `aqtinstall`, and thankfully that just works.